android/app/build.gradle

```diff
@@ -24,7 +24,7 @@ apply from: project(':react-native-config').projectDir.getPath() + "/dotenv.grad
 project.ext.react = [
     entryFile: "index.js",
     devDisabledInStaging: true,
-    enableHermes: false
+    enableHermes: true
```

Comment:
for some reason that was switch off - need find way

Reply:
It was disabled As it didn't work with RN 62 fir ios